app/views/content/home/_salaries-banner.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id="salary-banner" class="banner">
   <%= render CallsToAction::SimpleComponent.new(icon: "icon-money") do %>
-    <h2 class="call-to-action__heading">Earn at least &pound;30k a year</h2>
-    <p>All qualified teachers will have a <strong>starting salary of at least £30,000</strong> (or higher in London).</p>
+    <h2 class="call-to-action__heading">Earn a minimum of &pound;30k a year</h2>
+    <p>All qualified teachers get a <strong>minimum starting salary of £30k</strong> (or higher in London). So it pays to do what you love. And with lots of opportunities to progress, you could <strong>earn over £41k within 5 years</strong>.</p>
```

Comment:
```suggestion
    <p>All qualified teachers get a <strong>minimum starting salary of £30k</strong> (or higher in London), so it pays to do what you love. And with lots of opportunities to progress, you could <strong>earn over £41k within 5 years</strong>.</p>
```
Would this be better as one sentence? But happy for you to disagree

Reply:
Also - I wonder if the `And with lots of opportunities....` sentence might be better on a new line - might make the 30k and 41k stand out more?

Again, happy for you to disagree!